### DISCUSSION TITLE
14A19FE7-1296-480B-9215-D8EF13B233E4

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#228] [General]
# GLASS-JWST-BD1

GLASS-JWST-BD1 is a T8 brown dwarf located in the constellation Sculptor, 2055 light years away. It was the first brown dwarf discovered by the James Webb Space Telescope, in 2022.

https://celestia.mobi/addon?item=14A19FE7-1296-480B-9215-D8EF13B233E4

**Comment by denial9999** [comment]
She's like a gas gaint 🤨
